Command/ConsumeCommand.php

```diff
@@ -89,5 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             usleep($usleep);
         } while ($isLive());
+
+        return 0;
```

Comment:
Strangely, this bundle doesn't have a dependency with the console component. You could have taken advantage of the available constants `Command::SUCCESS` (and others exit code)

Reply:
Yep you're right, but as this bundle works also for version 4 of Symfony I cannot sadly use this const.

To be honest, this update it just for a transition between 2 Symfony versions before a migration to Symfony/Messenger which is more relevant